src/messages/request.rs

```diff
@@ -56,7 +56,7 @@ pub enum Request {
         /// Unique message identifier
         msg_id: MsgId,
         /// Requester public key
-        requester: sign::PublicKey,
+        requester: PublicKeys,
```

Comment:
Again, not correct - pls don't rampantly use PublicKeys instead of sign::PublicKey - we don't want to include the encryption keys uselessly in the transmitted data and bloat it. Precisely the point of Friday Gossip on safe_crypto